configure

```diff
@@ -632,6 +632,19 @@ fi
 
 echo >> configure.log
 
+# Check for -Werror=unguarded-availability-new compiler support
+echo "" > test.c
+  cat > $test.c <<EOF
+int main() { return 0; }
+EOF
+if $cc $CFLAGS -Werror=unguarded-availability-new -c $test.c >> configure.log 2>&1; then
+  echo "Checking for -Werror=unguarded-availability-new... Yes." | tee -a configure.log
+  ADDITIONAL_CHECK_FLAGS="-Werror=unguarded-availability-new"
+else
+  echo "Checking for -Werror=unguarded-availability-new... No." | tee -a configure.log
+  ADDITIONAL_CHECK_FLAGS=""
+fi
+
```

Comment:
Same as with cmake, should check for `-Werror=unguarded-availability`.

Reply:
Fixed